package.json

```diff
@@ -24,6 +24,26 @@
     "jquery-ui-dist": "~1.12",
     "photoswipe": "^4.1.1",
     "simple-ajax-uploader": "^2.5.5",
-    "slick-carousel": "^1.6.0"
+    "slick-carousel": "^1.6.0",
+    "standard": "^10.0.3"
+  },
+  "standard": {
```

Comment:
does this mean that when gulp runs it will check all the js?
Is there a way to also include this in the tests for travis?

Reply:
@mathiashelin can you still answer this question

Reply:
For the moment it does not, but it could be implemented later on.

Reply:
so how do we keep the files from diverting from the standard for now?
because if there is no way to automatically do that I think it should be added before we merge this one